packages/agoric-cli/README.md

```diff
@@ -2,37 +2,31 @@
 
 ## Prerequisites
 
-### Vagrant
+### Use as part of Agoric SDK
 
-To run a standardized Linux distribution with all the required development tools, you probably want [Vagrant](https://www.vagrantup.com/docs/):
+If you have cloned the Agoric SDK, you can use the Agoric CLI directly.
 
 ```sh
-vagrant up --provider=docker
-# or
-vagrant up --provider=virtualbox
-# then
-vagrant ssh
+export AGORIC="$PWD/bin/agoric --sdk"
```

Comment:
Does this need to be done from within `packages/agoric-cli`? If so, let's mention it.

Reply:
Yes, it should be.